internal/services/bot/bot_channel_directline_resource.go

```diff
@@ -310,6 +353,26 @@ func flattenDirectlineSites(input []botservice.DirectLineSite) []interface{} {
 			site["name"] = *v
 		}
 
+		if v := element.AppID; v != nil {
```

Comment:
We might want to change the style of this flattening function to always set all the properties, no matter they are nil or not, i.e. 

```go
var appId string
if v := element.AppID; v != nil {
  appId = *v
}
site["application_id"] = appId
```
We can do this either in this PR or in another PR.

Reply:
Updated